addons/storyshots/storyshots-core/src/frameworks/angular/renderTree.ts

```diff
@@ -5,7 +5,7 @@ import { TestBed } from '@angular/core/testing';
 // eslint-disable-next-line import/no-extraneous-dependencies
 import { BrowserDynamicTestingModule } from '@angular/platform-browser-dynamic/testing';
 import { addSerializer } from 'jest-specific-snapshot';
-import { RenderNgAppService } from '@storybook/angular';
+import { RenderNgAppService } from '@storybook/angular/renderer';
```

Comment:
@ThibaudAV trying this as something less aggressive 😄

Reply:
yes but I can see that it doesn't seem simple 💪 . Tell me if you need help with this PR